src/NuGet.Clients/NuGet.PackageManagement.UI/Models/ItemFilter.cs

```diff
@@ -5,6 +5,9 @@ namespace NuGet.PackageManagement.UI
 {
     public enum ItemFilter
     {
+        /// <summary>
+        /// Represents Browse tab in PM UI
```

Comment:
This might make it clearer that the comment is specifically for the All value.

```suggestion
        /// The value All represents the Browse tab in PM UI
```

Reply:
Thank you for the feedback. Fixed in the latest commit.